docs/writing-docs/autodocs.md

```diff
@@ -98,25 +123,44 @@ Storybook provides support for MDX 2, an open standard markup language, combinin
 
 <!-- prettier-ignore-end -->
 
-Create an MDX file for your component in the same directory as your stories and components to add your custom documentation. We recommend naming the file similar to your component (e.g., `Button.mdx`) for easy reference.
+Create an MDX file to add your custom documentation. Depending on how you want your documentation to render in the UI, you'll need to consider the following use cases.
+
+### Using the `Meta` Doc Block
```

Comment:
If this page is now titled _Automatic documentation and Storybook_, I don't understand why this entire section (_Setup custom documentation_) is located here and not as part of the _MDX_ page.

All of these examples demonstrate how to write custom MDX docs entries and have nothing to do with autoDocs.

Reply:
Can you elaborate on this a bit more? Are you suggesting moving the documentation to the MDX page starting at "Setup custom documentation" all the way to "Advanced configuration"? Is that what you're thinking?

Reply:
Yes, that's exactly what I'm suggesting. Everything in that section of content pertains to writing MDX documentation in general, not the automatic docs that Storybook provides.

I do think it'd be a good idea to point from here to there, though, explaining that if customizing the autoDocs page isn't enough, they can author their own custom MDX content.